src/plugin/nerd_plugin.js

```diff
@@ -213,6 +213,16 @@ class NerdPluginVirtual extends EventEmitter {
   }
 
   getFulfillment (transferId) {
+    return this._getFulfillment(transferId).then((fulfillment) => {
+      if (!fulfillment) {
+        return Promise.reject(null)
```

Comment:
Shouldn't it check if there's a transfer with that ID first, reject if not, then check whether there's a fulfillment and resolve to `null` if not?